similarity/config/configurations.properties

```diff
@@ -0,0 +1,20 @@
+wordMetric	paragram
```

Comment:
This is more like a preference. I'd suggest making this a configurator files extending `Configurator` class. 
@mssammon can probably correct if he doesn't agree with this.